tikv-client/src/main/java/com/pingcap/tikv/operation/iterator/DAGIterator.java

```diff
@@ -218,6 +238,20 @@ private SelectResponse process(RangeSplitter.RegionTask regionTask) {
     return SelectResponse.newBuilder().addAllChunks(resultChunk).setEncodeType(encodeType).build();
   }
 
+  private Store getTiFlashStore(TiRegion region) {
```

Comment:
what if there are two replicas for tiflash? shall we return a `List<Store>`?

Reply:
DONE.

Reply:
This will be addressed in another PR. It is about load balancing.